client/app/vps/translations/Messages_pt_PT.xml

```diff
@@ -238,7 +238,7 @@
    <translation id="vps_configuration_activate_ftpbackup_title_button" qtlid="209853">Ativar o Backup storage</translation>
    <translation id="vps_configuration_activate_ftpbackup_title" qtlid="209866">Ativar a opção Backup storage</translation>
    <translation id="vps_configuration_activate_ftpbackup_step1_question" qtlid="452605">Deseja ativar a opção Backup Storage para o VPS <span class="normal italic">{{ serviceName }}</span>?</translation>
-   <translation id="vps_configuration_activate_ftpbackup_step1_alert" qtlid="452618">Este opção terá um preço mensal de <span class="red bold">{{ price }} IVA não incluído</span></translation>
+   <translation id="vps_configuration_activate_ftpbackup_step1_alert" qtlid="452618">Este opção terá um preço mensal de <span class="bold">{{ price }} IVA não incluído</span></translation>
```

Comment:
Why change the portuguese translations instead of the english/french ones ?

Reply:
According to requirement, the price in the this particular translation, should not be in red. It was red in all languages and this was removed for all languages.

Reply:
I think we should split the translation to avoid html tags in it.